File: NapakalakiRuby/pkg/NapakalakiRuby-0.0.1/lib/prize.rb
#encoding: utf-8
#Autor: Dunia Rodríguez y Emilio Reguero
module NapakalakiGame
class Prize
  attr_reader :treasures, :level
  
  def initialize(treasures,level)
    @treasures = treasures
    @level = level
  end
  
  def getLevels
    @level
  end
  def to_s
    "Tesoros ganados: #{@treasures} \n Niveles ganados: #{@level}"
  end
end
end

File: Napakalaki/src/NapakalakiGame/Prize.java
/*
* PDOO Practicas - Napakalaki Java
* @author Emilio Reguero y Dunia Rodríguez 
*/

package NapakalakiGame;


public class Prize {
    private int level;
    private int treasures;
    
    
    public Prize(int treasures, int level){
        this.treasures = treasures;
        this.level = level;
    }
    
    
    int getTreasures(){
        return treasures;
    }
    
    int getLevel(){
        return level;
    }
    
    @Override
    public String toString(){
        return ("Treasures = " + Integer.toString(treasures) + " levels = " + Integer.toString(level));
    }
}
